File: official/docs/c#/shipments/CreateShipmentWithOptions.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Address to = await Address.Create(toAddress);
Address from = await Address.Create(fromAddress);
Parcel parcel = await Parcel.Create(parcel);

Shipment shipment = await Shipment.Create(new Dictionary<string, object>() {
  { "parcel", new Dictionary<string, object>() {
    {"id", "prcl_..."}
  } },
  { "to_address", new Dictionary<string, object>(){
    {"id", "adr_..."}
  } },
  { "from_address", new Dictionary<string, object>(){
    {"id", "adr_..."}
  }},
  { "options", new Dictionary<string, object>() {
     { "address_validation_level", 0 }
  } }
});

File: official/docs/python/shipments/create_shipment_with_options.py
import easypost
easypost.api_key = "EASYPOST_API_KEY"

shipment = easypost.Shipment.create(
    to_address={
        "id": "adr_..."
    },
    from_address={
        "id": "adr_..."
    },
    parcel={
        "id": "prcl_..."
    },
    options={
        'address_validation_level': 0
    }
)

print(shipment)
